src/macaron/slsa_analyzer/checks/provenance_l3_check.py

```diff
@@ -27,6 +29,27 @@
 logger: logging.Logger = logging.getLogger(__name__)
 
 
+class _VerifyArtefactResultType(Enum):
+    """Result of attempting to verify an asset."""
+
+    PASSED = "verify passed"
+    FAILED = "verify failed"
+    ERROR = "verify error"
```

Comment:
Can you please add comments for these three types to clarify how they are concluded/differentiated?

Reply:
This is addressed in [78b65b2](https://github.com/oracle-samples/macaron/pull/29/commits/78b65b23f3b61676ad407205fc9d7bee39fa7b99)